site/cypress/e2e/Cypress-Feature/late_days_cache.spec.js

```diff
@@ -44,21 +43,20 @@ const checkStudentsInCache = () => {
     for (const user_id of all_user_ids) {
         // Gradeable # of late days used should be empty
         cy.get(`[USER_ID="${user_id}"] > [id="Late Allowed Homework"]`)
-            .then((cell) => expect(cell.text().trim()).to.equal(''));
+            .then(cell => expect(cell.text().trim()).to.equal(''));
 
         // Remaining late days isnt known, should be empty
         cy.get(`[USER_ID="${user_id}"] > [ld_id="Late Days Remaining"]`)
-            .then((cell) => expect(cell.text().trim()).to.equal(''));
```

Comment:
Is there a reason that some of the then() instances use parenthesis and some of them don't? (similar with .each())

Reply:
The reason is that those were auto fixed, and eslint's linting rules don't really care about whether there are parentheses or not. I can look into enabling an additional rule that would force parentheses or not, but so far I've tried to make the config as close as possible to the original.

Reply:
The ones that had parentheses were changed to not, and the ones that didn't have any were changed to have them.

Reply:
I think 'always' would make everything more consistent, but I'd ask @williamjallen for his opinion.

Reply:
Personally I like 'always' more too. I'll wait for him to comment.

Reply:
"Always" seems reasonable to me.